scripts/create.mjs

```diff
@@ -43,10 +43,13 @@ const pascalCasedName = toPascalCase(recipe); // PascalCased recipe ID only cont
 (async () => {
   // Folder paths
   const userData =
-    process.env.APPDATA ||
-    (process.platform === 'darwin'
+  process.env.APPDATA || (
```

Comment:
Please do not use nested ternary syntax. Please change to either `switch-case` statement or `if-else` conditions.

Reply:
But... It works!

Reply:
the question is not only about it working, its also about understabability/maintainability

Reply:
What do you mean? Those who are not familiar with it will not look at the source code or try to set it up! One should have a certain amount of experience if one want to work in this area! And if one have that...

Reply:
But please, have added notes! I think that should be enough.

Reply:
maintainability